src/handle-data.ts

```diff
@@ -79,7 +82,11 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
   } = info;
   const isHidden = qError?.qErrorCode === 7005;
   const isLocked = isDim && (info as ExtendedNxDimensionInfo).qLocked;
-  const autoAlign = isDim ? 'left' : 'right';
+  const { qDimensionType } = info as ExtendedNxDimensionInfo;
+  const autoHeadCellTextAlign = qDimensionType === 'N' || qDimensionType === undefined ? 'right' : 'left';
```

Comment:
I remember that we had some table use case where column was showing some sort of image not numbers of strings. 

My questions is what we will get `qDimensionType` in those cases and will those be automatically right aligned?